packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -1,278 +1,317 @@
-                  <Text component={TextVariants.h2}>Lay it out</Text>
-                  <Text component={TextVariants.p} className="ws-mdx-p">
-                    A layout provides a fully responsive page structure that keeps your components organized and aligned regardless of screen size.
-                  </Text>
-                  <Text
-                    className="pf-v5-u-pl-0 pf4-c-link__text ws-mdx-a"
-                  >
-                    <Link to="/layouts/bullseye" aria-label="view layouts">
-                      View layouts <ArrowRightIcon />
-                    </Link>
-                  </Text>
-                </div>
-                <div className="pf-v5-u-pb-2xl">
-                  <Text component={TextVariants.h2}>Build things your way</Text>
-                  <Text component={TextVariants.p} className="ws-mdx-p">
-                    Components are like building blocks. Designed to be flexible and modular, you can mix and match components to create a solution for almost any UI problem.
-                  </Text>
-                  <Text
-                    className="pf-v5-u-pl-0 pf4-c-link__text ws-mdx-a"
-                  >
-                    <Link to="/components/about-modal" aria-label="view components">
-                      View components <ArrowRightIcon />
-                    </Link>
-                  </Text>
-                </div>
-                <div className="pf-v5-u-pb-2xl">
-                  <Text component={TextVariants.h2}>See it in action</Text>
-                  <Text component={TextVariants.p} className="ws-mdx-p">
-                    Components are better together. Demos show how multiple components can be used in a single design. They also provide  some useful starter code, so the days of creating complex layouts from scratch are over.
-                  </Text>
-                  <Text
-                    className="pf-v5-u-pl-0 pf4-c-link__text ws-mdx-a"
-                  >
-                    <Link to="/patterns/primary-detail" aria-label="view demos">
-                      View demos <ArrowRightIcon />
-                    </Link>
-                  </Text>
-                </div>
-              </TextContent>
-            </GridItem>
-            <GridItem sm={12} md={6}>
-              <div className="pf4-c-image__laptop" aria-hidden />
-              <div className="pf4-c-image__phone" aria-hidden />
-              <div className="pf4-c-image__screen" aria-hidden />
-              <div className="pf4-c-image__desktop" aria-hidden />
-            </GridItem>
-          </Grid>
-        </GridItem>
+    <PageSection isWidthLimited>
+      <Grid hasGutter md={6} xl={3}>
+        <Card isFullHeight>
+          <CardHeader
+            style={{
+              minHeight: "200px",
+              backgroundPosition: "center",
+              backgroundSize: "cover",
+              backgroundImage: `url(${featuredPostsData.post1.imageURL})`,
+              height: "300px",
```

Comment:
Good points @mcoker - I moved the CSS and updated the full cards to be clickable, that definitely makes sense. 

Using `<img>` tags here is a great idea too but setting images to the full width of the `CardHeader` doesn't seem to work with selectable/clickable PF cards, and using background image is the workaround for this spacing issue: https://codesandbox.io/s/szrpwr?file=/index.tsx:1880-2033 ... we could open up follow up issues to address this if you think this would be a common use case?